composer.json

```diff
@@ -19,5 +19,13 @@
         "psr-4": {
             "PHPUnitRetry\\": "src/"
         }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "PHPUnitRetry\\Tests\\": "tests/"
+        }
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^8.0"
```

Comment:
This is causing the tests to fail, since PHPUnit 8 does not support PHP 7.1. This is part of the reason why I was using `composer require phpunit/phpunit`, since `phpunit ^7` is required on PHP 7.1.

We could update this to be `"phpunit/phpunit": "^7.0|^8.0", which will fix the failing tests.